project_indicators/README.rst

```diff
@@ -0,0 +1,48 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Project Indicators
+=================================================
+
+
+You can track your all the project status with the respective tasks list.
+Also you can print the Project Tracking Report in PDF format.
+
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/project-reporting/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/project-reporting/issues/new?body=module:%20project_billing_utils%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+
```

Comment:
Kindly add usage section so it will help end user to check on runbot.
https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst